src/main/resources/interface-specification.yml

```
@@ -836,11 +844,15 @@ components:
       properties:
         version:
           type: string
+          minLength: 10
+          maxLength: 250
         answers:
           additionalProperties:
             type: array
             items:
               type: string
+              minLength: 10
```

Our enum values may be shorter

```suggestion
              minLength: 1
```